src/main/java/se/sundsvall/mex/businesslogic/worker/UpdatePhaseWorker.java

```diff
@@ -10,24 +10,24 @@
 import org.zalando.problem.Problem;
 import org.zalando.problem.Status;
 
-import se.sundsvall.workflow.businesslogic.handler.FailureHandler;
+import se.sundsvall.mex.businesslogic.handler.FailureHandler;
 
 @Component
-@ExternalTaskSubscription("myWorker")
-public class MyWorker implements ExternalTaskHandler {
+@ExternalTaskSubscription("updatePhaseTask")
+public class UpdatePhaseWorker implements ExternalTaskHandler {
 
-	private static final Logger LOGGER = LoggerFactory.getLogger(MyWorker.class);
+	private static final Logger LOGGER = LoggerFactory.getLogger(UpdatePhaseWorker.class);
 
 	private final FailureHandler failureHandler;
 
-	public MyWorker(FailureHandler failureHandler) {
+	public UpdatePhaseWorker(FailureHandler failureHandler) {
 		this.failureHandler = failureHandler;
 	}
 
 	@Override
 	public void execute(ExternalTask externalTask, ExternalTaskService externalTaskService) {
 		try {
-			LOGGER.info("Execute My Worker");
+			LOGGER.info("Execute task for updating phase");
 			if ("throw_exception".equals(externalTask.getBusinessKey())) {
 				throw Problem.valueOf(Status.I_AM_A_TEAPOT, "Big and stout");
```

Comment:
Remove?

Reply:
This is just a skeleton model/worker setup. It will be removed/replaced in later tasks.